experiment/experiment.py

```diff
@@ -286,6 +286,21 @@ def get_case_vals(self, case):
         """
         return self._case_vals[case]
 
+    def get_file_fieldcases(self, field, **case_kws):
+        """ Return a list with the string of filepath and filename
+        associated with a particular case and field.
+
+        Parameters
+        ----------
+        field : str
+            The name of the field to match files for.
+        case_kws: dict
+            The dictionary of a particular set of key values for cases from this
+            experiment.
+
+        """
+        return [fil for case, fil in self.walk_files(field) if case_kws == case] 
```

Comment:
The only change I'd make is to tweak this to 

``` python
return [file for case, file in self.walk_files(field) if case_kws == case] 
```

just because I don't think there's a need to shorten a 4-letter English word :)